man/systemd-sbsign.xml

```diff
@@ -85,11 +99,44 @@
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--signed-attributes=<replaceable>PATH</replaceable></option></term>
+        <term><option>--signed-attributes-signature=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>Configure the signed attributes blob (as provided by
+        <command>export-signed-attributes</command>) and corresponding signature for the
+        <command>sign</command> command.</para>
+
+        <xi:include href="version-info.xml" xpointer="v258"/></listitem>
+      </varlistentry>
+
       <xi:include href="standard-options.xml" xpointer="help"/>
       <xi:include href="standard-options.xml" xpointer="version"/>
     </variablelist>
   </refsect1>
 
+
+  <refsect1>
+    <title>Examples</title>
+
+    <example>
+      <title>Offline EFI secure boot signing of a PE binary</title>
+
+      <para>The following does offline secure boot signing of systemd-boot:</para>
+
+      <programlisting>SD_BOOT="$(find /usr/lib/systemd/boot/efi/ -name "systemd-boot*.efi" | head -n1)"
+/usr/lib/systemd/systemd-sbsign export-signed-attributes --certificate cert.crt --output sattrs.bin "$SD_BOOT"
+openssl dgst -sha256 -sign privkey.pem -out sattrs.sig sattrs.bin
+/usr/lib/systemd/systemd-sbsign \
+    sign \
+    --certificate cert.crt \
+    --output "$SD_BOOT.signed" \
+    --signed-attributes sattrs.bin \
+    --signed-attributes-signature sattrs.sig \
+    "$SD_BOOT"</programlisting>
+    </example>
+  </refsect1>
```

Comment:
can you break up the example in three parts, clarifying that the first line is on the signing client, the second line is supposed to be invoked on the signing server, and the third line is on the client again?

(or alternatively insert some shell style `#` comments briefly clarifying that)